### DISCUSSION TITLE
Desktop everything works fine, but Web3Modal not popping up on Mobile.

### DISCUSSION THREAD

**Discussion by CaioRutka** [WalletConnect/walletconnect-monorepo#4358] [Developer Support] [answered]
Hello,

I'm current using web3modal v4, just implement with ethers. 
Did everything according to the docs, but my modal don't pop up on my mobile device. 
On the desktop, works fine, but when i access via Mobile, and click on Connect, nothing happens.

Att,

Caio Rutka

**Comment by glitch-txs** [comment, marked as answer]
HI, could you share a minimal reproducible example?

You can see some of our examples here: https://github.com/WalletConnect/web-examples/tree/main/dapps/web3modal

**Reply by randomrandom** [reply]
I have the same issue and saw this thread which describes the problem quite well: https://github.com/orgs/WalletConnect/discussions/4478

**Comment by joshi-kaushal** [comment]
I have a similar issue with RainbowKit. Were you able to solve it?
